README.md

```diff
@@ -108,6 +102,64 @@ etcd cluster by adding the following to the Helm install command:
 --set etcd.enabled=false --set "operator.controllerSet.spec.dbConnectionURL=jdbc:postgresql://postgres/postgresdb?user=postgresadmin&password=admin123"
 ```
 
+### Configuring storage pool creation
+The piraeus operator installed by helm can be used to create storage pools. Creation is under control of the
+LinstorNodeSet resource:
+```
+$ kubectl get LinstorNodeSet.piraeus.linbit.com piraeus-op-ns -o yaml                                                                                       [24/1880]
+kind: LinstorNodeSet
+metadata:
+..
+spec:
+  ..
+  storagePools:
+    lvmPools:
+    - name: lvm-thick
+      volumeGroup: drbdpool
+    lvmThinPools:
+    - name: lvm-thin
+      thinVolume: thinpool
+      volumeGroup: drbdpool
+
+```
+
+There are two ways to configure storage pools
+
+#### On install
+On install time, by setting the value of `operator.nodeSet.spec.storagePools` when running helm install.
+
+First create a file with the storage configuration like:
+```yaml
+operator:
+nodeSet:
+  spec:
+    storagePools:
+      lvmPools:
+      - name: lvm-thick
+        volumeGroup: drbdpool
+      ..
+```
+This file can be passed to the helm installation like this:
+```
+helm install -f <file> charts/piraeus-op
+```
+
+#### After install
+On a cluster with the operator already configured (after helm install)
+
+* Get the current configuration:
+```
+$ kubectl get LinstorNodeSet.piraeus.linbit.com <nodesetname> -o yaml > nodeset.yaml
```

Comment:
Does `kubectl edit` not work here? It makes the instructions shorter and anyone with kubernetes experience will know that they can do it this way if they prefer.